res/TensorFlowLiteRecipes/BroadcastTo_002/test.recipe

```diff
@@ -0,0 +1,25 @@
+operand {
+  name: "bc_input"
+  type: FLOAT32
+  shape { dim: 2 dim: 3 }
+}
+operand {
+  name: "bc_shape"
+  type: INT32
+  shape { dim: 3 }
+  filler { tag: "explicit" arg: "1" arg: "2" arg: "3" }
+}
+operand {
+  name: "bc_ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 2 dim: 3 }
+}
+operation {
+  type: "BroadcastTo"
+  input: "bc_input"
+  input: "bc_shape"
+  output: "bc_ofm"
+  version: 3
```

Comment:
why is this 3?

Reply:
Do we have any conditional branch or test with version is 2 or 3 ?
if not, I don't think we have to add version 3.

Reply:
Or, we can add the check if the version is 2 or 3 in the `validate` function of `compiler/luci/import/src/Nodes/CircleBroadcastTo.cpp`.

```c++
bool CircleBroadcastToGraphBuilder::validate(const ValidateArgs &args) const
{
  auto const bt_version = args.reader.opcodes().at(args.op.opcode_index)->version();
  // The version one of broadcast to op won't be not supported
  // https://github.com/tensorflow/tensorflow/blob/ce2d218fc88c20b1d81abb017eabd436779865e4/tensorflow/lite/kernels/register.cc#L306-L311
  if (bt_version != 2 && bt_version != 3)
    return false;

  return GraphBuilder::validate(args, 2);
}
```

Reply:
As I know, for all other Ops we didn't do any version checking: min, max values.
I don't know our IR has version... -_-;
For now, let's skip this version check in import.

Reply:
Okay. Then how about use version 3? I'm not sure but latest version might be good..?

Reply:
> ... but latest version might be good..?

Found that circle IR doesn't have version thus doesn't import version per Op.
The version may effect tflite interpreter for value check.
Any version is OK to me with working value check.